pkgs/applications/science/electronics/openboardview/default.nix

```diff
@@ -38,8 +45,13 @@ stdenv.mkDerivation rec {
 
   dontWrapGApps = true;
   postFixup = ''
-    wrapGApp "$out/bin/${pname}" \
-      --prefix LD_LIBRARY_PATH : ${lib.makeLibraryPath [ gtk3 ]}
+    if [ -e "$out/openboardview.app" ]; then
```

Comment:
To avoid rebuilds on Linux:

```nix
  postFixup = lib.optionalString stdenv.isDarwin ''
    ...
  '' + lib.optionalString (!stdenv.isDarwin) ''
    ...
  '';
```